Source/FolioReaderFontsMenu.swift

```diff
@@ -243,7 +243,12 @@ class FolioReaderFontsMenu: UIViewController, SMSegmentViewDelegate, UIGestureRe
             scrollDirection = readerConfig.scrollDirection
         }
 
-        layoutDirection.selectSegmentAtIndex(scrollDirection?.rawValue ?? 0)
```

Comment:
Are you sure that this is the cause of the crash? Because it fallback to 0 if is nil.